tests/dem/tests_particle_particle_contact_force.h

```diff
@@ -0,0 +1,90 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025 The Lethe Authors
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+
+#ifndef tests_particle_particle_contact_force_h
+#define tests_particle_particle_contact_force_h
+
```

Comment:
I would rename that file to just be like particle_helper_function or test_particles_functions
Something similar that kinda says the name a bit explicitely.